File: Sum of Odd Numbers up to N/SumOfOddNumbersUptoN.java
public class SumOfOddNumbersUptoN {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int total = 0;
        for (int i = 1; i <= n; i++) {
            if (i % 2 != 0) {
                total = total + i;
            }
        }
        System.out.println("Sum of odd numbers from 1 to " + n + " is: " + total);
    }
}

File: Sum of Odd Numbers up to N/sum_of_odd_numbers_upto_n.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    int total = 0;
    for (int i = 1; i <= n; i++) {
        if (i % 2 != 0) {
            total = total + i;
        }
    }
    printf("Sum of odd numbers from 1 to %d is: %d\n", n, total);
    return 0;
}
